tools/c7n_azure/tests/cassettes/StorageTest.test_storage_settings_require_secure_transfer.json

```diff
@@ -0,0 +1,789 @@
+{
+    "version": 1,
+    "interactions": [
+        {
+            "request": {
+                "method": "GET",
+                "uri": "https://management.azure.com/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/providers/Microsoft.Storage/storageAccounts?api-version=2019-04-01",
```

Comment:
Can you remove the storage accounts that you don't need for the test from the JSON response?